pages/common/yt-dlp.md

```diff
@@ -22,16 +22,16 @@
 
 - Specify audio format and audio quality of extracted audio (between 0 (best) and 10 (worst), default = 5):
 
-`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{url}}"`
```

Comment:
Thanks for your contribution. Can you update the same in other examples?

Reply:
What other examples?

Reply:
> What other examples?

The placeholder examples above.